src/components/MaterialUI/ScopeSelector/ScopeTreeView.js

```diff
@@ -2,14 +2,15 @@ import React from "react";
 import TreeViewMui from "@material-ui/lab/TreeView";
```

Comment:
Since TreeView is gone... This is a breaking change?

Reply:
It is used just in ScopeSelector - which is new Component rewrited to MUI and it is also updated in this PR